### PR TITLE
NMS-12105: Changed AS to type long

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-12105

Please double check whether this will break any existing installation 
since the Elasticsearch template was also changed.

### CODE REVIEW

File: features/telemetry/protocols/netflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/ipfix/SrcDstAsTestIpFix.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        return bsonDocument;
+    }
+
+    @Test
+    public void testSrcDstAs() {
+        long value1 = 2147483648L;
+        long value2 = 2147483649L;
+        final BsonDocument bson = getBsonDocument("bgpSourceAsNumber", value1, "bgpDestinationAsNumber", value2);
+        final IpfixFlow flow = new IpfixFlow(bson);
+        Assert.assertTrue(flow.getSrcAs() > 0);
```
Comment (j-white) [inline features/telemetry/protocols/netflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/ipfix/SrcDstAsTestIpFix.java]: Given that we're actually verifying the exact value bellow, checking that it's > 0 seems redudant.

File: features/telemetry/protocols/sflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/sflow/adapter/SrcDstAsTestSFLow.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        bsonDocumentWriter.writeEndDocument();
+        return bsonDocument;
+    }
+
+    @Test
+    public void testSrcDstAs() {
+        long value = 2147483648L;
+        final BsonDocument bson = getBsonDocument(value);
+        final SFlow flow = new SFlow(null, bson);
+        Assert.assertTrue(flow.getSrcAs() > 0);
```
Comment (j-white) [inline features/telemetry/protocols/sflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/sflow/adapter/SrcDstAsTestSFLow.java]: Similarly here.

File: features/telemetry/protocols/netflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/ipfix/SrcDstAsTestIpFix.java (hunk 1 above)
Comment (j-white) [reply features/telemetry/protocols/netflow/adapter/src/test/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/ipfix/SrcDstAsTestIpFix.java]: We should also avoid using `assertTrue` for verifying these types of conditions. `assertThat(flow.getSrcAs(), greaterThan(0))` would provide better feedback if the assertion were to fail.

**Comment by christianpape** [comment]
Still waiting for approval :-)

**Pull request merged**